code/modules/automation/auto_labeler.dm

```diff
@@ -0,0 +1,56 @@
+//Labels items like a hand labeler
+
+/obj/machinery/automation/label //Takes a recipe datum, accepts ingredients for them and makes it
```

Comment:
🤔

Reply:
copypasta

Otherwise works like a hand labeler, gives crates manifests that prints out all items inside of the crate at the time of labeling

Reply:
copypasta

Otherwise works like a hand labeler, gives crates manifests that prints out all items inside of the crate at the time of labeling

Reply:
copypasta

Otherwise works like a hand labeler, gives crates manifests that prints out all items inside of the crate at the time of labeling